packages/amplication-server/src/core/git/git.resolver.ts

```diff
@@ -16,21 +21,82 @@ export class GitResolver {
   constructor(private readonly gitService: GitService) {}
   //#region Mutation
   @Mutation(() => GitRepo)
-  @AuthorizeContext(AuthorizableResourceParameter.AppId, 'appId')
+  @AuthorizeContext(
+    AuthorizableResourceParameter.GitOrganizationId,
+    'gitOrganizationId'
+  )
   async createRepoInOrg(@Args() args: CreateRepoArgs): Promise<GitRepo> {
     return this.gitService.createRepo(args);
   }
+
+  @Mutation(() => GitOrganization)
+  @AuthorizeContext(
+    AuthorizableResourceParameter.GitOrganizationId,
+    'gitOrganizationId'
+  )
+  async getOrganization(@Args() args: BaseGitArgs): Promise<GitOrganization> {
+    return this.gitService.getGitOrganization(args);
+  }
+
+  @Mutation(() => GitOrganization)
+  //@AuthorizeContext(AuthorizableResourceParameter.WorkspaceId, 'workspaceId')
+  async createOrganization(
+    @Args() args: CreateGitOrganizationArgs
+  ): Promise<GitOrganization> {
+    return await this.gitService.createGitOrganization(args);
+  }
+
+  @Mutation(() => Boolean)
+  @AuthorizeContext(
+    AuthorizableResourceParameter.GitOrganizationId,
+    'gitOrganizationId'
+  )
+  async deleteOrganization(@Args() args: BaseGitArgs): Promise<boolean> {
+    return this.gitService.deleteGitOrganization(args);
+  }
+
+  @Mutation(() => AuthorizeAppWithGithubResult)
+  //@AuthorizeContext(AuthorizableResourceParameter.WorkspaceId, 'workspaceId')
+  async getGithubAppInstallationUrl(
+    @Args() args: GetGitInstallationUrlArgs
+  ): Promise<AuthorizeAppWithGithubResult> {
+    return {
+      url: await this.gitService.getGithubAppInstallationUrl(args)
+    };
+  }
+
   //#endregion
   //#region Query
   @Query(() => [GitRepo])
-  @AuthorizeContext(AuthorizableResourceParameter.AppId, 'appId')
+  @AuthorizeContext(
+    AuthorizableResourceParameter.GitOrganizationId,
+    'gitOrganizationId'
+  )
   async getReposOfUser(@Args() args: GetReposListArgs): Promise<GitRepo[]> {
```

Comment:
Either change the name as suggested below - or better - use a generic query called FindGitRepositories which accepts parameters from the client
also, the name of the args should be the same as the name of the query

also, can a user get a list of repos that are not belong to them? why to call it "OfUser"?

```suggestion
  async getGitReposOfUser(@Args() args: GetReposListArgs): Promise<GitRepo[]> {
```